File: ILT.IHR.APP/ClientApp/src/app/core/interfaces/TimeEntry.ts
export class ITimeEntry {
    TimeEntryID: number;
    TimeSheetID: number;
    Project: string;
    Activity: string;
    WorkDate: Date;
    Hours: number;
    CreatedBy: string;
    CreatedDate: Date;
    ModifiedBy: string;
    ModifiedDate: Date;
    TimeStamp: ['']
}
export class ITimeEntryDisplay {
    timeEntryID: number;
    timeSheetID: number;
    project: string;
    activity: string;
    workDate: Date;
    hours: number;
    createdBy: string;
    createdDate: Date;
    modifiedBy: string;
    modifiedDate: Date;
}

File: ILT.IHR.DTO/TimeEntry.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace ILT.IHR.DTO
{
    public class TimeEntry: AbstractDataObject
    {
        public int TimeEntryID
        {
            get { return base.RecordID; }
            set { base.RecordID = value; }
        }

        public int TimeSheetID { get; set; }
        [MaxLength(50)]
        public string Project { get; set; }
        [MaxLength(50)]
        public string Activity { get; set; }        
        public DateTime WorkDate { get; set; }        
        public int Hours { get; set; }
    }
}
